src/types/index.ts

```diff
@@ -111,6 +111,30 @@ export type AnonymizedAggregate = {
   anonValue: number | null;
 };
 
+// AnonymizationParams
+
+export type Interval = {
+  lower: number;
+  upper: number;
+};
+
+export type SuppressionParams = {
+  lowThreshold: number;
+  sD: number;
```

Comment:
`sD` is a weird name, just `sd` or `SD`?

Reply:
it's SD in F# world, but the auto JSON decoder with `CamelCase` strategy interprets `sD`. I didn't want to uproot the naming conventions _yet_ (e.g. expand this to `StdDev`) or introduce any special cases in the JSON codecs.